src/controllers/facilityController.js

```diff
@@ -26,5 +26,67 @@ const getFacilities=async(req, res)=>{
   }
  }
 
+ /**
+ * PATCH facility
+ * */
+ const updateFacility = async(req, res)=> { 
+    try {    
+      const _facility=await model.Facility.findOne({ where: { id:req.params.id } });
+      if(_facility){
+        const _facilityResult = await model.Facility.update(req.body, { where: { id: req.params.id}});
+        if(_facilityResult) {
+          const data=await model.Facility.findOne({ where: { id:req.params.id } });
+          res.status(200).send({
+            status: "Ok",
+            message: res.__('facility updated succeddfuly'),
+            data
+          });
+        }
+      }else{
+        res.status(404).send({
+          status: "bad",
```

Comment:
Instead of status, some of the team uses `success: true or false`, I think it will be good to adopt a convention across our codebase